plaso/output/shared_elastic.py

```diff
@@ -110,9 +110,15 @@ def _FlushEvents(self):
       # pylint does not recognizes request_timeout as a valid kwarg. According
```

Comment:
It looks like this comment might be no longer relevant, remove?

Reply:
Done